cloudinit/net/dhcp.py

```diff
@@ -293,12 +316,17 @@ def parse_static_routes(rfc3442):
         ("169.254.169.254/32", "130.56.248.255"), ("0.0.0.0/0", "130.56.240.1")
     ]
 
+    sr2 = parse_state_routes("24.191.168.128 192.168.128.1,0 192.168.128.1")
```

Comment:
Can you fix the typo in the function name on this line and line 314 at the same time?

Reply:
good catch ! fix coming.